modules/snooker/src/test/ts/browser/CopySelectedTest.ts

```diff
@@ -80,13 +80,20 @@ UnitTest.test('CopySelectedTest', () => {
     return table;
   };
 
-  const check = (label: string, expected: TestData[][], input: TestData[][]) => {
+  const check = (label: string, expected: TestData[][], input: TestData[][], tableAttributes?: { addBeforeCopy: Record<string, string | boolean | number>; removedAfterCopy: string[] }) => {
```

Comment:
Could you split this out into a separate type for the table attributes please?

Reply:
Also, `addBeforeCopy` is a bit of a confusing name. Maybe we should just call these `beforeCopy` and `afterCopy`?

Reply:
Fixed